roles/kubernetes/node/templates/kubelet-container.j2

```diff
@@ -11,6 +11,5 @@
 -v /var/lib/kubelet:/var/lib/kubelet \
```

Comment:
i'm pretty sure we need to add the following mounts:
/etc/ssl
{{etcd_config_dir}}

Maybe /run but I don't think it's needed